README.md

```diff
@@ -56,6 +57,13 @@ On Mac via Homebrew:
 brew install kind
 ```
 
+On Mac via MacPorts:
```

Comment:
this should also be in site/ quick-start.md's installation guide.

our contributing guide has details for contributing to the docs, rendering them etc. https://kind.sigs.k8s.io/docs/contributing/getting-started/

Reply:
I have refined the documentation for macOS installation to be more consistent with the `README.md` and `quick-start.md`.

Reply:
Should the Kind version be updated to `v0.11.0` within the `README.md` or should this remain at `v0.10.0` for now?